jetty-core/jetty-deploy/src/main/java/org/eclipse/jetty/deploy/providers/ContextProvider.java

```diff
@@ -317,6 +317,9 @@ else if (Supplier.class.isAssignableFrom(context.getClass()))
         if (context instanceof Deployable deployable)
             deployable.initializeDefaults(properties);
 
+        if (Boolean.valueOf(properties.get("graceful")))
```

Comment:
Use `parseBoolean()` instead of `valueOf()`.
However, I don't understand this?
`ContextHandler implements Graceful` and the properties likely won't have `graceful=true`, so it won't be set here.

I would have expected these lines to not be necessary and `ContextHandler` just be graceful.